images/miq-app/docker-assets/container-scripts/container-deploy-common.sh

```diff
@@ -113,6 +116,29 @@ fi
 
 }
 
+function check_if_new_replica() {
+# Description
+# Check for pre-existing server data into PV, if not found, we assume a new server/replica case
+# Always skip if we are performing a new_deployment
+
+echo "== Checking for existing data on server PV =="
+
+if [[ ! -d ${PV_CONTAINER_DATA_DIR} && ${DEPLOYMENT_STATUS} != new_deployment ]]; then
```

Comment:
Is it enough to check the directory only ? (I think it is created on the initialization of the logs ...
Shouldn't we validate the GUID is there (and just make sure the directories are there ?)
The GUID is the part that determines the appliance personality ...